common/src/objects/stored_workload_spec.rs

```diff
@@ -22,12 +22,14 @@ use crate::helpers::serialize_to_ordered_map;
 use super::{AddCondition, Tag, WorkloadInstanceName, WorkloadSpec};
 
 #[derive(Debug, Serialize, Default, Deserialize, Clone, PartialEq, Eq)]
-#[serde(default, rename_all = "camelCase")]
+#[serde(rename_all = "camelCase")]
```

Comment:
The third example from the [user documentation](https://eclipse-ankaios.github.io/ankaios/main/reference/complete-state/) does not work anymore. The one with `ank set state ...`. I guess it is due to this change. We shall do something with it I think. We shall not unsupported things in the user documentation.